src/Plugin/CookiePlugin.php

```diff
@@ -91,15 +91,13 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     /**
      * Creates a cookie from a string.
      *
-     * @param $setCookie
-     *
      * @return Cookie|null
      *
      * @throws TransferException
      */
-    private function createCookie(RequestInterface $request, $setCookie)
+    private function createCookie(RequestInterface $request, string $setCookieHeader)
```

Comment:
`: ?Cookie`?

Reply:
I guess we'd need PHP 7.1 for that. Is there any reason why 2.x is php 7.0+ and not 7.1+. 7.0 is EOL already.

Reply:
thats why i did not do it. but i wondered the same. happy to move to 7.1+. wdyt @Nyholm ?

Reply:
I agree. Lets do PHP7.1

Reply:
cool! however, i'd prefer to do that in a separate pull request, as it will entail quite a lot of changes.